hack/testdata/cass-cluster-test.template.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: "navigator.jetstack.io/v1alpha1"
+kind: "CassandraCluster"
+metadata:
+  name: "test"
```

Comment:
This should be a CASS_NAME parameter I think

Reply:
:+1: